istio/src/test/scala/io/buoyant/k8s/istio/ApiserverClientTest.scala

```diff
@@ -6,7 +6,7 @@ import com.twitter.finagle.http.{Request, Response}
 import com.twitter.util.Future
 import io.buoyant.k8s.istio.ApiserverClient.RouteRuleConfig
 import io.buoyant.test.{Awaits, Exceptions}
-import istio.proxy.v1.config.{DestinationWeight, RouteRule}
+import _root_.istio.proxy.v1.config.{DestinationWeight, RouteRule}
```

Comment:
what is the _root_ prefix?

Reply:
Intellij being silly with protobuf imports -_- fixed